frontend/src/routes/Authentication/Login/index.tsx

```diff
@@ -108,14 +108,14 @@ export default function Login() {
   const [isLoading, setIsLoading] = useState(false)
   const [username, setUsername] = useState('')
   const [password, setPassword] = useState('')
-  const passwordHash = sha256(password).toString()
 
   const [error, setError] = useState({ message: '' })
 
   const loginHandler = async () => {
     if (username !== '' && password !== '') {
```

Comment:
if (username && password) works, I think.

Reply:
fixed in b107d423c52f6872e1ac8f532f0be309b285cba0